src/coins.h

```diff
@@ -153,6 +158,13 @@ struct CCoinsCacheEntry
     };
 
     CCoinsCacheEntry() noexcept = default;
+
+    // No implicit copying, only moves.
+    CCoinsCacheEntry(CCoinsCacheEntry&&) noexcept = default;
+    CCoinsCacheEntry& operator=(CCoinsCacheEntry&&) noexcept = default;
+    CCoinsCacheEntry(const CCoinsCacheEntry&) noexcept = delete;
+    CCoinsCacheEntry& operator=(const CCoinsCacheEntry&) noexcept = delete;
```

Comment:
nit: deleted functions don't need `noexcept`

Reply:
[Pushed](https://github.com/bitcoin/bitcoin/compare/cd400acd8360144b8c7daa4fd2c6bea8267f9ba9..db91623c5241b3795a45940cff0b6e79be6f265f), thank you